src/PublicApiGenerator/CecilEx.cs

```diff
@@ -58,41 +58,33 @@ public static bool IsCompilerGenerated(this IMemberDefinition m)
             return m.CustomAttributes.Any(a => a.AttributeType.FullName == "System.Runtime.CompilerServices.CompilerGeneratedAttribute");
         }
 
-        public static bool HasVisiblePropertyMethod(this MemberAttributes attributes)
-        {
-            var access = attributes & MemberAttributes.AccessMask;
-            return access == MemberAttributes.Public ||
-                   access == MemberAttributes.Family ||
-                   access == MemberAttributes.FamilyOrAssembly;
-        }
-
-        public static MemberAttributes GetPropertyAttributes(MemberAttributes getterAttributes, MemberAttributes setterAttributes)
```

Comment:
Thinking out loud. Should this be an extension method on the first param because it is placed in an extension file?

Reply:
Not sure I know the answer, good question.

Reply:
Technically, there is no concept of an _extension file_ or even an _extension class_. There is an _extension method_. And it can be located in any static class. In addition, even the class itself is not called Cecil**Extensions**.